docs/index.rst

```diff
@@ -1,31 +1,56 @@
-.. TabPy documentation master file, created by
-   sphinx-quickstart on Tue Mar 12 01:49:50 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
-Welcome to TabPy's documentation!
-=================================
-
-.. toctree::
-   :maxdepth: 2
-   :caption: Contents:
-
-   about.md
-   server-config.md
-   server-configurations.md
-   server-download.md
-   server-rest.md
-   server-startup.md
-   tabpy-tools.md
-   tabpy-virtualenv.md
-   TableauConfiguration.md
-   security.md
-
-
-
-Indices and tables
-==================
-
-* :ref:`genindex`
-* :ref:`modindex`
-* :ref:`search`
+TabPy
+=====
+
+TabPy (Tableau Python Server) is external server implementation which
+allows expanding Tableau with executing Python scripts on table
+calculation.
+
+
+ .. toctree::
+   :maxdepth: 2
+   :caption: General documentation:
+
+-  `About TabPy <about.md>`__
+-  `TabPy Server Download Instructions <server-download.md>`__
+-  `TabPy Server Configuration Instructions <server-config.md>`__
+-  `TabPy Server Startup Instructions <server-startup.md>`__
+-  `Supported TabPy Server
+   Configurations <server-configurations.md>`__
+-  `Running TabPy in Python Virtual
+   Environment <tabpy-virtualenv.md>`__
+-  `Authoring Python calculations in
+   Tableau <TableauConfiguration.md>`__.
+-  `TabPy Tools <tabpy-tools.md>`__
+
+ .. toctree::

```

Comment:
So rather than having multiple toctree directives like this (not sure how well that is supposed) might want to consider just creating subfolders in docs to house the various topics. You can then link to an `index.rst` within each subfolder to get the same effect with better organization